Cargo.toml

```diff
@@ -36,6 +36,9 @@ members = [
     "megazords/lockbox",
     "testing/sync-test",
     "tools/protobuf-gen",
+
+    "examples/*/",
+    "testing/separated/*/",
```

Comment:
Is this going to interact badly with the changes over in https://github.com/mozilla/application-services/pull/3329?